docs/static/monitoring/collectors.asciidoc

```diff
@@ -3,11 +3,11 @@
 [[logstash-monitoring-collectors]]
```

Comment:
I'm still seeing the xpack rendering in the preview
<img width="753" alt="Screen Shot 2020-04-09 at 11 35 08 AM" src="https://user-images.githubusercontent.com/4061337/78912969-4ca6d700-7a56-11ea-90f1-55f7bcb6be81.png">

Reply:
Are you speaking of the badge in the banner?  It is supposed to be there for now.  The X-Pack badge is a link that takes users to our licensing page. 

We are removing extra instances of X-Pack from the text.